Yoast/YoastSEO.js
# Red bullet for titles wider than 600 pixels

## Summary

This PR can be summarized in the following changelog entry:

* Titles wider than 600 pixels will result in a red bullet from now on. 

## Relevant technical choices:

* I added the max length and the actual length to the assessment results objects for the metaDescriptionLengthAssessment and pageTitleWidthAssessment. This is needed for https://github.com/Yoast/wordpress-seo/issues/9588.

## Test instructions

This PR can be tested by following these steps:

* Link this branch to wordpress-sep.
* Use the (old) desktop snippet preview in the classic editor.
* Write a title wider than 600px (=when the progress bar is completely filled).
* You should see a red bullet and the following feedback text: The SEO title is wider than the viewable limit. Before, the bullet was orange.

Connect https://github.com/Yoast/wordpress-seo/issues/9588
